numpy/lib/tests/test_function_base.py

```diff
@@ -997,25 +999,26 @@ def test_slices(self):
                     self._check_inverse_of_slicing(s)
 
     def test_fancy(self):
+        a = self._create_arrays()[0]
```

Comment:
Total nit, but I find this a little bit easier to read on the first pass - my eyes often skip the array indexing:

```suggestion
        a, _ = self._create_arrays()
```